### PR TITLE
Add JCB credit card type

### PR DESCRIPTION
**Pull request by mihaileu**
Adding support for JCB credit card

### CODE REVIEW

**Comment by pimjansen** [comment]
There are changes done that should not be there. Also the build is failing @mihaileu

**Comment by mihaileu** [comment]
Yeah I see. It was initially forked from original repo and somehow contains more code. I'll come back with the proper PR's as seems the old ones are not ok

**Comment by pimjansen** [comment]
@mihaileu will you update these PRs or whats the plan?

**Comment by mihaileu** [comment]
Yes, not now but today I hope.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

File: src/Faker/Provider/Payment.php [hunk 1]
```diff
@@ -146,9 +151,10 @@ public static function creditCardType()
     /**
      * Returns the String of a credit card number.
      *
-     * @param string  $type      Supporting any of 'Visa', 'MasterCard', 'American Express', and 'Discover'
-     * @param boolean $formatted Set to true if the output string should contain one separator every 4 digits
-     * @param string  $separator Separator string for formatting card number. Defaults to dash (-).
+     * @param string $type      Supporting any of 'Visa', 'MasterCard', 'American Express', and 'Discover'
```
Comment (chris-doehring) [inline src/Faker/Provider/Payment.php:154]:
```suggestion
     * @param string $type      Supporting any of 'Visa', 'MasterCard', 'American Express', 'Discover' and 'JCB'
```

**Review by chris-doehring — CHANGES_REQUESTED** [review]

**Comment by Nyholm** [comment]
Could you please revert the unrelated CS changes.

**Comment by bramceulemans** [comment]
There seem to be conflicts. Can you please rebase onto main and revert the code style changes (`array()` to `[ ]`, etc.)?

**Pull request closed without merging**